src/plugins/embeddable/public/lib/embeddables/create_embeddable_factory.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Maybe this file could be called `default_embeddable_factory_provider.ts`, so looking at file system it would be clear what is inside.